web/src/components/molecules/Asset/Asset/AssetBody/svg-preview.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from "@emotion/styled";
+import { useCallback, useEffect, useState } from "react";
+
+type SVGPreviewProps = { url: string; svgRender: boolean };
+
+const SVGPreview: React.FC<SVGPreviewProps> = ({ url, svgRender }) => {
+  const [svgText, setSvgText] = useState<string>("");
+
+  const fetchData = useCallback(() => {
+    fetch(url)
+      .then((res) => res.text())
+      .then((val) => {
+        setSvgText(val);
+      });
+  }, [url]);
+
+  useEffect(() => {
+    fetchData();
+  }, [fetchData]);
+
+  return (
+    <>{svgRender ? <Image src={url} alt="svg-preview" /> : <p>{svgText}</p>}</>
+  );
```

Comment:
You don't need the fragments. Just double check locally that this syntax is correct and to fix formatting!
```suggestion
  return {svgRender ? (<Image src={url} alt="svg-preview" />) : (<p>{svgText}</p>)}
  );
```